src/base/solvers/solveCobraLP.m

```diff
@@ -38,6 +38,10 @@
 %                   without regularisation.
 %
 %    primalOnly:    {(0), 1}; 1 = only return the primal vector (lindo solvers)
+%   
+%    solverParams:  solver-specific parameter structure. Formats supported
+%                   are ILOG cplex and Tomlab parameter syntax. see example
+%                   for details.
```

Comment:
This comment says it works for ILOG and Tomlab (shouldn't this be restricted to tomlab, as it is only used there?
it's also already mentioned as "parameters" below (which should serve the same purpose)